### DISCUSSION TITLE
How to transfer accounts from Google authenticator official android app to chrome authenticator extension?

### DISCUSSION THREAD

**Discussion by sundarsrst** [Authenticator-Extension/Authenticator#637] [Q&A]
There is a update feature news in Overview:
- Import data from Google Authenticator offical mobile App

Steps followed:
1. Opened authenticator app in phone.
2. clicked transfer accounts.
3. could see QR code 1 and QR code 2.
4. clicked scan button in extension in chrome, scanned both but says "unrecognized QR codes"

is this the right way to - "Import data from Google Authenticator offical mobile App"

**Comment by ira-at-work** [comment]
for some reason I could not get the QR code scanned either (with a screenshot, not sure how you got it to the extension). I had to export from Google Auth, scan it with a QR reader (used Google lens, very irresponsible, but still possibly more trusted than other options unless you have one guaranteed to work offline, blocked from sending your scans to a third party), copied the decyphered text (the android clipboard, yet another risk) and pasted it in the extention's import screen, seems to have worked. had to remove a few duplicates of accounts I already had, but generally it worked OK.

**Reply by Sneezry** [reply]
Google official Authenticator mobile App allows you to export up to 10 accounts one time in a single big QR image. The QR image contains too many details and the extension may fail to recognize the code correctly. You can try to export 5 accounts a time and the extension should work as expected.
